help/code-blocks.md

```diff
@@ -67,15 +67,23 @@ prefix**.
 {end_tabs}
 
 For example, to configure code playgrounds for languages like Python or
-JavaScript, you could specify the language and URL prefix fields as:
+JavaScript, you could specify the language and URL templates as:
 
-* `Python` and `https://replit.com/languages/python3/?code=`
-* `JavaScript` and `https://replit.com/languages/javascript/?code=`
+* `Python` and `https://replit.com/languages/python3/{?code}`
+* `JavaScript` and `https://replit.com/languages/javascript/{?code}`
 
 When a code block is labeled as Python or JavaScript (either explicitly or by
 organization default), users would get a on-hover option to open the code block
 in the specified code playground.
 
+!!! tip ""
+
+    Code playgrounds use [RFC 6570](https://www.rfc-editor.org/rfc/rfc6570.html)
+    compliant URL templates to describe how links should be generated. `{?code}` is an
+    expression for a URL query parameter whose value come from the variable `code`.
```

Comment:
So I think this is a pretty confusing example for use with a website that happens to use something other than `code`  for their URLs (I'm pretty sure there 's some variety); I think for that reason we want `https://replit.com/languages/python3/?code={code}` as the example rather than using `{?code}` which makes an assumption about what the third-party website has as their format, and to explain something like "Zulip's rendering engine will pass the URL-encoded code from the code block as the `code` parameter to this URL template in order to generate the URL". maybe with a bit more editing.

Reply:
Updated this part of the documentation:

    Code playgrounds use [RFC 6570](https://www.rfc-editor.org/rfc/rfc6570.html)
    compliant URL templates to describe how links should be generated. Zulip's
    rendering engine will pass the URL-encoded code from the code block as the
    `code` parameter, denoted as `{code}` in this URL template, in order to
    generate the URL. You can refer to parts of the documentation on URL
    templates from [adding a custom linkifier](/help/add-a-custom-linkifier).

with an edit noting that `code` refers to `{code}`, the RHS of the query parameter.